File: LeetCode-Java/Length of Last Word.java
public class Solution {
    public int lengthOfLastWord(String s) {
        int index = s.length() - 1;
        int result = 0;
        while(index >= 0 && s.charAt(index) == ' ')
            index --;
        while(index >= 0)
        {
            if(s.charAt(index) == ' ')
                break;
            result ++;
            index --;
        }
        return result;
    }
}

File: Leetcode-cpp/length_of_last_word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int last_index = s.length() - 1;
        while (last_index >= 0 && s[last_index] == ' ') last_index--;
        int first_index = last_index;
        while (first_index >= 0 && s[first_index] != ' ') first_index--;
        return last_index - first_index;
    }
};